infra/logger_test.go

```diff
@@ -0,0 +1,34 @@
+package infra_test
```

Comment:
it's generally good to have more comprehensive UTs, however i don't think this logger test is necessary. It's simply testing `logrus`, which is **not our pkg**